python/lsst/scarlet/lite/utils.py

```diff
@@ -48,9 +48,9 @@ def integrated_gaussian_value(x: np.ndarray, sigma: float) -> np.ndarray:
     gaussian:
         A Gaussian function integrated over `x`
     """
-    lhs = erfc((0.5 - x) / (sqrt2 * sigma))
-    rhs = erfc((2 * x + 1) / (2 * sqrt2 * sigma))
-    return np.sqrt(np.pi / 2) * sigma * (1 - lhs + 1 - rhs)
+    lhs = erfc((x - 0.5) / (sqrt2 * sigma))
+    rhs = erfc((x + 0.5) / (sqrt2 * sigma))
+    return np.sqrt(np.pi) * 0.5 * sigma * (lhs - rhs)
```

Comment:
Might as well define `sqrt(pi)` if you have `sqrt2`. Then again, I'm guessing you don't call this often enough to warrant just pybinding it and `integrated_circular_gaussian` instead.

Reply:
Meh, why not. I only use it in one place but it will get called a lot.